cmd/openshift-install/gather.go

```diff
@@ -95,30 +96,30 @@ func runGatherBootstrapCmd(directory string) error {
 	if err != nil {
 		if err2, ok := err.(errUnSupportedGatherPlatform); ok {
 			logrus.Error(err2)
-			return unSupportedPlatformGather()
+			return unSupportedPlatformGather(directory)
 		}
 		return errors.Wrapf(err, "failed to get bootstrap and control plane host addresses from %q", tfStateFilePath)
 	}
 
-	logGatherBootstrap(bootstrap, port, masters)
+	logGatherBootstrap(bootstrap, port, masters, directory)
 	return nil
 }
 
-func logGatherBootstrap(bootstrap string, port int, masters []string) {
-	var (
-		sshport string
-		scpport string
-	)
-	if port != 22 {
-		sshport = fmt.Sprintf(" -p %d", port)
-		scpport = fmt.Sprintf(" -P %d", port)
+func logGatherBootstrap(bootstrap string, port int, masters []string, directory string) {
+	logrus.Info("Pulling logs from bootstrap for debugging")
+
+	client, err := ssh.NewClient("core", fmt.Sprintf("%s:%d", bootstrap, port), nil)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	if err := ssh.Run(client, fmt.Sprintf("/usr/local/bin/installer-gather.sh %s", strings.Join(masters, " "))); err != nil {
+		logrus.Fatal(err)
 	}
-	if s, ok := os.LookupEnv("SSH_AUTH_SOCK"); !ok || s == "" {
-		logrus.Info("Make sure ssh-agent is running, env SSH_AUTH_SOCK is set to the ssh-agent's UNIX socket and your private key is added to the agent.")
+	lFile := filepath.Join(directory, "log-bundle.tar.gz")
```

Comment:
nit: `lFile` -> `file`?  There aren't other file variables in this function to distinguish from, right?

Reply:
done https://github.com/openshift/installer/pull/1822/commits/cad7f02a8bdfc268ee54ab4f4e25dec1a99640c3